cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -336,9 +350,7 @@ def original_request_class(appliance):
 
 @pytest.fixture(scope="module")
 def modified_request_class(request, domain, original_request_class):
-    with pytest.raises(Exception, match="error: Error during 'Automate Class copy'"):
-        # methods of this class might have been copied by other fixture, so this error can occur
-        original_request_class.copy_to(domain)
+    original_request_class.copy_to(domain)
     klass = (domain
```

Comment:
@jaryn 
- We have [domain](https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/automate.py#L6) as global fixture.
- So you can [remove this local fixture of ```domain``` form this file](https://github.com/ManageIQ/integration_tests/pull/8620/files#diff-fc91ed43845421f3eda7a5bb17217dc7R335) and use global one.